components/normalizer/benches/data/TestNames_Japanese_h.txt

```diff
@@ -0,0 +1,50 @@
+かげやま,みのる
```

Comment:
Nit: Add a README file for how these files were generated so that someone can add or re-generate them in the future